packages/default-theme/components/SwProductCard.vue

```diff
@@ -59,6 +62,16 @@ export default {
     }
   },
   computed: {
+    getName() {
```

Comment:
This definitely should be placed inside product helpers :)

Reply:
done